sys-fs/btrfsmaintenance/files/btrfsmaintenance-0.5.1-script-fix.patch

```diff
@@ -0,0 +1,11 @@
+--- a/btrfsmaintenance-functions
```

Comment:
This is missing a PR link, which I'll add: https://github.com/kdave/btrfsmaintenance/pull/119.